projects/js-packages/components/components/contextual-upgrade-trigger/types.ts

```diff
@@ -0,0 +1,5 @@
+export type CutBaseProps = {
+	description: string;
+	cta: string;
+	onClick: () => void;
```

Comment:
There is an in-built `VoidFunction` for this.

```suggestion
	onClick: VoidFunction;
```